docs/reference/requests/lookup_accounts.md

```diff
@@ -10,6 +10,10 @@ transfer. Instead, set the
 accounts to limit their account balances. More complex conditional transfers can be expressed using
 [balance-conditional transfers](../../coding/recipes/balance-conditional-transfers.md).
 
+⚠️ It is not possible currently to look up more than a full batch (8190) of accounts atomically.
+When issuing multiple `lookup_accounts` calls, it can happen that other operations will interleave
+between the calls leading to read skew.
```

Comment:
Maybe reference the `history` flag? Seems like in a use case where you need >1 batch of atomic accounts that might be useful.